### PR TITLE
Release 2.4.0

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: news/_posts/2015-09-30-akka-2.4.0-released.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+The artifacts comprising this release have been published to https://oss.sonatype.org/content/repositories/releases/ and also to Maven Central. In addition, we adopted the sbt standard of encoding the Scala binary version in the artifact name, i.e. the core actor package’s artifactId is “akka-actor_2.11” or “akka-actor_2.12.0-M2”, respectively.
+
+357 issues were closed since 2.3.0. The complete list can be found in the [2.4-M1](https://github.com/akka/akka/issues?q=milestone%3A2.4-M1), [2.4-M2](https://github.com/akka/akka/issues?q=milestone%3A2.4-M2), [2.4-M3](https://github.com/akka/akka/issues?q=milestone%3A2.4-M3), [2.4.0-RC1](https://github.com/akka/akka/issues?q=milestone%3A2.4.0-RC1), [2.4.0-RC2](https://github.com/akka/akka/issues?q=milestone%3A2.4.0-RC2), [2.4.0-RC3](https://github.com/akka/akka/issues?q=milestone%3A2.4.0-RC3) and [2.4.0](https://github.com/akka/akka/issues?q=milestone%3A2.4.0) github issues milestones.
+
+#### Credits ####
+
+Thanks to the Community! For this release we had the help of 134 committers.
+
+    commits added removed
+      287   84638   36405 Patrik Nordwall
```
Comment (ktoso) [inline news/_posts/2015-09-30-akka-2.4.0-released.md]: Add 3 spaces? ;) 
// that's my formatting OCD showing

Comment (patriknw) [inline news/_posts/2015-09-30-akka-2.4.0-released.md]: good catch, it's actually a tab

Comment (ktoso) [inline news/_posts/2015-09-30-akka-2.4.0-released.md]: :smile:

**Comment by ktoso** [comment]
LGTM, thanks :)

File: 404.html [hunk 2]
```diff
@@ -69,31 +69,13 @@ <h2 id='documentation'>Documentation</h2>
 
 <ul>
 
-<li>Akka 2.3.12 Documentation - <a href='http://doc.akka.io/docs/akka/2.3.12/'>http://doc.akka.io/docs/akka/2.3.12/</a> (or as PDF for <a href='http://doc.akka.io/docs/akka/2.3.12/AkkaJava.pdf'>Java</a> or <a href='http://doc.akka.io/docs/akka/2.3.12/AkkaScala.pdf'>Scala</a> and also in <a href='http://doc.akka.io/docs/akka/2.3.12/Akka.epub'>EPUB format</a>)</li>
+<li>Akka Documentation - <a href='http://doc.akka.io/docs/akka/current/'>http://doc.akka.io/docs/akka/current/</a> (or as PDF for <a href='http://doc.akka.io/docs/akka/current/AkkaJava.pdf'>Java</a> or <a href='http://doc.akka.io/docs/akka/current/AkkaScala.pdf'>Scala</a> and also in <a href='http://doc.akka.io/docs/akka/current/Akka.epub'>EPUB format</a>)</li>
 
-<li>Akka 2.3.12 API - <a href='http://doc.akka.io/api/akka/2.3.12'>for Scala</a>&nbsp;and&nbsp;<a href='http://doc.akka.io/japi/akka/2.3.12'>for Java</a></li>
```
Comment (drewhk) [inline 404.html]: nice cleanup, 404 only needs current.

**Comment by drewhk** [comment]
LGTM

**Pull request merged**